lib/matplotlib/tests/test_animation.py

```diff
@@ -167,17 +136,74 @@ def animate(i):
         line.set_data(x, y)
         return line,
 
+    # "fixture" can be passed to determine the class returned by the fixture
```

Comment:
I was a bit confused by the name, since `fixture` means something else (ish) in pytest; can we name it differently? Maybe like `class_constructor` or `return_type` or something shorter that I can't think of.

Reply:
I went with `klass` because it's very short, I've seen it used a lot in `scikit-learn`, and it at least shows up a few times in `matplotlib` according to `git grep`.